frontend/src/stores/userStore.ts

```diff
@@ -18,14 +18,16 @@ export const useUserStore = defineStore('users', {
 		me: {} as User,
 		errors: [] as String[],
 		// persists data accross refreshes
-		isLoggedIn: useStorage("isLoggedIn", false, sessionStorage),
+		isLoggedIn: useStorage('isLoggedIn', false, sessionStorage),
 	}),
 
 	// getters == computed values
-	getters: {
-	},
+	getters: {},
 	// actions == methods
 	actions: {
+		// this should be moved out of the userStore
+		// it is not userStore functionality
+		// and it should be typed with something other than any..
```

Comment:
yes, should probably make a ticket out of this.. (created a draft ticket for now)

Reply:
This specifically handles the errors array for the userStore right? Should we mak it a util that returns an array of errors?

Reply:
but don't we have forms on other pages as well?
We could make a "formStore"? not sure about this anymore...

Reply:
We can maybe make it a helper function that returns the array of errors? This way it can be used everywhere but the errors are kept wherever you want to use them?

Reply:
Feel free to resolve this after reading this